integration_tests/tests/tasks/allocations.cy.ts

```diff
@@ -18,210 +18,219 @@ import { fullPersonFactory } from '../../../server/testutils/factories/person'
 import { signIn } from '../signIn'
 
 context('Task Allocation', () => {
-  beforeEach(() => {
-    cy.task('reset')
-    cy.task('stubSignIn')
-
-    const cruManagementArea = cruManagementAreaFactory.build()
-    const cruManagementAreas = [...cruManagementAreaFactory.buildList(4), cruManagementArea]
-    const qualifications = qualificationFactory.buildList(2)
-    // Given there are some users in the database
-    const users = [...userWithWorkloadFactory.buildList(3), userWithWorkloadFactory.build({ qualifications })]
-    const selectedUser = users[0]
-
-    // And there is an allocated task
-    const applicantUserDetails = applicationUserDetailsFactory.build()
-    const caseManagerUserDetails = applicationUserDetailsFactory.build()
-    const application = applicationFactory.build({
-      person: fullPersonFactory.build(),
-      applicantUserDetails,
-      caseManagerUserDetails,
-      caseManagerIsNotApplicant: true,
-      apType: 'pipe',
-      isWomensApplication: false,
-    })
-
-    const task = taskFactory.build({
-      allocatedToStaffMember: userFactory.build(),
-      applicationId: application.id,
-      taskType: 'Assessment',
-      probationDeliveryUnit: { id: '2', name: 'East Sussex (includes Brighton and Hove)' },
-    })
-
-    const restrictedPerson = personFactory.build({ isRestricted: true })
-    const applicationForRestrictedPerson = applicationFactory.withReleaseDate().build({
-      apType: 'pipe',
-      person: restrictedPerson,
-      isWomensApplication: false,
-    })
+  const cruManagementArea = cruManagementAreaFactory.build()
+
+  const cruManagementAreas = [...cruManagementAreaFactory.buildList(4), cruManagementArea]
+  const qualifications = qualificationFactory.buildList(2)
+  // Given there are some users in the database
```

Comment:
Missed a `GIVEN` and an `AND` 😄

Reply:
Ah no, they are outside the test 'it()' so a `cy.log` without the context, blows up!